SQL/New_patches/2021-07-28_diagnosis_evolution.sql

```diff
@@ -0,0 +1,25 @@
+-- TODO: Update schema
+CREATE TABLE `diagnosis_evolution` (
+  `DxEvolutionID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) DEFAULT NULL,
+  `ProjectID` int(10) unsigned DEFAULT NULL,
+  `visitLabel` varchar(255) DEFAULT NULL,
+  `instrumentName` varchar(255) DEFAULT NULL,
+  `sourceField` varchar(255) DEFAULT NULL,
+  `orderNumber` int(10) unsigned DEFAULT NULL,
+  PRIMARY KEY (`DxEvolutionID`),
+  CONSTRAINT `FK_DxEvolution_1` FOREIGN KEY (`ProjectID`) REFERENCES `Project` (`ProjectID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `candidate_latest_diagnosis` (
+  `ID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `CandID` int(6) NOT NULL,
+  `ProjectID` int(10) unsigned NOT NULL,
```

Comment:
i don't think ProjectID needs to be in this table? it can be queried from diagnosis_evolution via the DxEvolutionID, not sure if i'm missing something though